Exam/CC1/src/exercice3/Dictonary.java

```diff
@@ -0,0 +1,22 @@
+package exercice3;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Dictonary {
+
+	Map<String, String> list = new HashMap<>();
+	public void add(Word w) {
+		list.put(w.getId(), w.getDefinition());
+	}
+	public Word find(String id) {
+			if(list.get(id) != null) {
```

Comment:
See comment above...
This whole block could be simplified as `return list.get(id)` if you had stored `Word` in `list` instead of `String`